src/main/java/com/springweb/dv_spring_web_mongo/model/Project.java

```diff
@@ -3,44 +3,26 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.springweb.dv_spring_web_mongo.DTO.ProjectDTO;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
 
 import javax.annotation.processing.Generated;
 
+@Data
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
```

Comment:
When you add constructor to POJO object please provide constructor without parameters too.

Reply:
```
@NoArgsConstructor
@AllArgsConstructor
```
it is enough because `access = AccessLevel.PUBLIC` will be by default